dts/bindings/can/nxp,s32-canxl.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP S32 CANXL controller
+
+compatible: "nxp,s32-canxl"
+
+include: ["can-fd-controller.yaml", "pinctrl-device.yaml"]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  clock-frequency:
+    type: int
+    required: true
+    description: Module clock frequency in Hz.
```

Comment:
Can this frequency be determined using the clock controller driver instead? Or is that not available for the S32 (yet)?

Reply:
currently, the clock controller driver isn't available for NXP S32